qa/L0_large_payload/large_payload_test.py

```diff
@@ -45,7 +45,7 @@ def setUp(self):
         # except the Python and plan backend should be able to handle payloads
```

Comment:
Run formatter (if you have not yet) and fix copyright line

Reply:
Will do.

Reply:
Updated.